data/sql/updates/pending_db_world/culling_of_stratholme.sql

```diff
@@ -1,7 +1,7 @@
-﻿DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9610') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9611') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='9612') AND (`OptionID`='1');
-DELETE FROM `gossip_menu_option` WHERE (`MenuID`='11277') AND (`OptionID`='1');
+﻿DELETE FROM `gossip_menu_option` WHERE `MenuID`=9610 AND `OptionID`=1;
```

Comment:
```suggestion
﻿DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (9610, 9611, 9612, 11277) AND `OptionID`=1;
```
https://www.azerothcore.org/wiki/sql-standards#compact-queries